src/frameworks/react-i18next.ts

```diff
@@ -0,0 +1,39 @@
+import { I18nextFramework } from './i18next'
+import { LanguageId } from '~/utils'
+
+class ReactI18nextFramework extends Framework {
```

Comment:
This PR does not seem to have been checked at all. Surely `ReactI18nextFramework` should be extending `I18nextFramework` and not `Framework`. `I18nextFramework` is also a default export and not a named one...